R/nhs_selectInput.R

```diff
@@ -1,33 +1,57 @@
-#' nhs_selectInput Function
+#' nhs_selectInput
 #'
-#' @importFrom shiny tagList
+#' NHS styled selectInput
+#'
+#' @param inputId Element id
+#' @param label Element label
+#' @param choices Character vector (optionally named) giving available choices
+#' @param selected Initially selected choices
+#' @param full_width Boolean, controls if full width or not
+#' @param ... Further named args passed to \code{shiny::selectInput}
+#'
+#' @return HTML
+#' @export
+#'
+#' @examples
+#' nhs_selectInput(
+#'   "fruit",
+#'   "Choose some fruit",
+#'   choices = c("Apple", "Banana", "Cherry"),
+#'   selected = "Banana",
+#'   full_width = TRUE
+#' )
 nhs_selectInput <- function(inputId,
                             label,
                             choices,
                             selected = NULL,
-                            full_width = FALSE) {
+                            full_width = FALSE,
+                            ...) {
```

Comment:
Added the `...` so it is possible to pass further named arguments to `shiny::selectInput`